mypy_django_plugin/lib/helpers.py

```diff
@@ -182,6 +183,12 @@ def get_call_argument_by_name(ctx: Union[FunctionContext, MethodContext], name:
     Return the expression for the specific argument.
     This helper should only be used with non-star arguments.
     """
+    # try and pull the named argument from the caller first
+    for kinds, argnames, args in zip(ctx.arg_kinds, ctx.arg_names, ctx.args):
+        for kind, argname, arg in zip(kinds, argnames, args):
+            if kind == ArgKind.ARG_NAMED and argname == name:
+                return arg
```

Comment:
one could imagine a perverse class that does something like:

```python
class FK(ForeignKey[_ST, _GT]):
   def __init__(self, *args, **kwargs):
       kwargs['null'] = not kwargs.get('null', False)
       super().__init__(*args, **kwargs)
```

but I'm thinking that's probably not going to happen and possibly ok to ignore?

Reply:
I guess they could even do that today and trick the current code with:

```python
class FK(ForeignKey[_ST, _GT]):
    def __init__(self, *args, null: bool = False, **kwargs):
        null = not null
        super().__init__(*args, null=null, **kwargs)
```

so definitely not a problem imo :)

Reply:
What do you think about reusing `_get_argument` from `mypy.plugins.common`? Would it solve this problem?

Reply:
I think the current code is equivalent to that so it wouldn't help (_get_argument only looks at callee args)